packages/core/src/StarknetWindowObject.ts

```diff
@@ -1,10 +1,55 @@
-import type { AccountInterface, ProviderInterface } from "starknet"
-import type {
-  AccountInterface as AccountInterfaceV4,
-  ProviderInterface as ProviderInterfaceV4,
-} from "starknet4"
+type FELT = string
 
-export type AccountChangeEventHandler = (accounts: string[]) => void
+type Call = {
```

Comment:
@avimak 
Maintaining type duplication might cause issues upon type change in starknetjs.
Why not taking those types from starknetjs (while installing starknetjs as dev dependency and import types only)?

Reply:
I also agree to this. Maybe we should publish only types package of starknet.js from CI under `@types/starknet` and use as a dev dependency here. Other projects might also benefit from these types. @ivpavici wdyt?

Reply:
this was mentioned iirc on the last call and indeed we could do it
cc: @naorye2 do you want to help with this? We have some budget for external contributors :)

Reply:
@ivpavici 
I generated locally type definitions from the openrpc json files and currently trying those types with snjs (build & tests). Hopefully will update soon.
Anyway I think that for the sake of this PR it is better to import types (only!) from snjs (as dev dependency) and change it when the final solution will be implemented.

Reply:
ok, I'll do it - import snjs as dev deps for its types

Reply:
after discussing with @naorye2 and Toni, we decided that since the fundamental idea here is to completely decouple get-starknet from snjs, it doesn't make sense to limit its API to snjs known/supported APIs, hence we should align only on the RPC API types; a complete alignment of both snjs and get-starknet on RPC API generated types is planned for 01/24